### DISCUSSION TITLE
Native support for radius 0

### DISCUSSION THREAD

**Discussion by marcopiii** [buildo/bento-design-system#394] [Ideas]
Currently, the theme supports the radius `4`, `8`, and `16`.
If you need a custom border-radius in your project you can easily extend the native sprinkles to introduce the `0` case.
However, it seems a reasonably typical case so maybe it should be supported by the native theme.

**Comment by gabro** [comment]
@Phyele what do you think? I think it's fair to allow `0` as a radius value for all Bento projects, but I wanted to confirm this with you

**Reply by gabro** [reply]
discussed offline with @Phyele, it's correct that Bento supports it, so I'll add it
